apps/docs/.vitepress/config.ts

```diff
@@ -51,8 +51,8 @@ export default defineConfig({
             link: '/guide/introduction/getting-started',
           },
           {
-            text: 'Quickstart',
-            link: '/guide/introduction/quickstart',
+            text: 'Creating a Fuel dApp',
+            link: '/guide/creating-a-fuel-dapp/',
```

Comment:
```suggestion
            text: 'Creating a Fuel dApp',
            link: '/guide/introduction/creating-a-fuel-dapp',
```

I believe moving this to the introduction folder will pass the validation.
To pass the validation from [here](https://github.com/FuelLabs/github-actions/blob/master/docs-hub/vp-docs.js).

Reply:
Why so? It seems to be complaining about the file name for some reason. Not where the file is placed

Reply:
The issue lies [here](https://github.com/FuelLabs/github-actions/blob/master/docs-hub/vp-docs.js#L134C1-L135C1), where we're not replacing all the spaces for hyphens (for a folder name).

<img width="935" alt="image" src="https://github.com/FuelLabs/fuels-ts/assets/16990131/15216b91-6ee6-4102-9fe9-e331652af6a5">

---

By moving the doc, we can bypass this logic and trigger a functional [replacement](https://github.com/FuelLabs/github-actions/blob/master/docs-hub/vp-docs.js#L167).

(This is for sure a hack, but thought I'd share the knowledge)

Reply:
Oh that's annoying. Thanks for pointing this out and sharing the solution!